pkg/yurthub/proxy/remote/loadbalancer.go

```diff
@@ -175,6 +177,12 @@ func (lb *loadBalancer) IsHealthy() bool {
 	return false
 }
 
+func (lb *loadBalancer) UpdateLastKubeletLeaseReqTime(lastKubeletLeaseReqTime time.Time) {
+	for i := range lb.backends {
+		lb.backends[i].checker.UpdateLastKubeletLeaseReqTime(lastKubeletLeaseReqTime)
```

Comment:
It's not need to set `lastKubeletLeaseReqTime` for every backend, and maybe we can use checker to call `UpdateLastKubeletLeaseReqTime` only once.

btw: we need add a new filed `checker` for `loadBalancer`

Reply:
please skip the above comment. 
we don't need to add `UpdateLastKubeletLeaseReqTime()` func for `loadBalancer`. and we can add a new field `checker` for `yurtReverseProxy`, so we can call `UpdateLastKubeletLeaseReqTime` directly in `ServeHTTP`

Reply:
Updated